spec/menu_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'menu'
+
+describe Menu do
+
+  let(:biriyani){double :biriyani, title: "Biriyani", price: 5.00}
+
+  context 'when created' do 
+    it 'has a default set of dishes' do
+      expect(subject.dishes).to eq Menu::DEFAULT_MENU
+    end
+
+    xit 'displays itself'
+      # Don't know how to test for this
```

Comment:
Try adding a `print` method to `Menu` (that you'll probably want to extract to a `Printer` class) and test that method